searchcore/src/vespa/searchcore/proton/server/resource_usage_explorer.cpp

```diff
@@ -47,6 +47,8 @@ ResourceUsageExplorer::get_state(const vespalib::slime::Inserter &inserter, bool
         memory.setDouble("utilization", usageState.memoryState().utilization());
         memory.setLong("physicalMemory", _usageFilter.getHwInfo().memory().sizeBytes());
         convertMemoryStatsToSlime(_usageFilter.getMemoryStats(), memory.setObject("stats"));
+        size_t transient_memory = _usageFilter.get_transient_memory_usage();
+        memory.setLong("transient", transient_memory);
```

Comment:
Consider adding metric that exposes the transient memory usage in ResourceUsageMetrics.

Reply:
Done